pygame/mask.py

```diff
@@ -1,6 +1,6 @@
 # Implement the pygame API for the bitmask functions
 
-from math import atan2, pi
+import math
```

Comment:
If this is intended to import from `pygame.math` it should use an explicit relative import, otherwise we should probably for `from __future__ import absolute_import` to avoid confusion.